packages/config/lib/options.ts

```diff
@@ -116,11 +128,20 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     name: 'env',
     defaultValue: {},
     validation: validate.isPlainObject,
+    isWriteable: true,
   }, {
     name: 'execTimeout',
     defaultValue: 60000,
     validation: validate.isNumber,
-  }, {
+    isWriteable: true,
+  },
+  {
+    name: 'exit',
```

Comment:
Not sure about this being writeable. If you set to `false` at runtime, does it actually keep a run from exiting?

Reply:
This is where it is being changed by `Cypress.config()`: https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress/integration/commands/debugging_spec.js#L107

Reply:
Right, but that's just for our own tests and it works there for testing out `cy.pause()`. Setting it won't actually keep Cypress from exiting, so it shouldn't be publicly writeable.

Reply:
Ah that makes more sense- I'll add this to the list of ones to check for internal use. Thanks!